aptos-move/framework/aptos-framework/sources/account/common_account_abstractions/derivable_account_abstraction_solana.move

```diff
@@ -0,0 +1,249 @@
+/// Domain account abstraction using ed25519 hex for signing.
+///
+/// Authentication takes digest, converts to hex (prefixed with 0x, with lowercase letters),
+/// and then expects that to be signed.
+/// authenticator is expected to be signature: vector<u8>
+/// account_identity is raw public_key.
+module aptos_framework::derivable_account_abstraction_ed25519_hex {
```

Comment:
old module name :)

this is a nit but maybe daa_sign_in_with_solana is better? feel free to leave as derivable_account_abstraction_solana if you prefer.

Reply:
can even be more specific `derivable_account_abstraction_phantom_solana`, as different wallets might need different DAA implementations